db/models/applications.js

```diff
@@ -372,6 +372,17 @@ class ApplicationEmailQueue extends Model {
                 type: DataTypes.UUID
             },
 
+            /**
+             * User model reference
+             * (optional as it was added after the model was created)
+             * Intended to allow looking up a user's deleted applications
+             * after the application itself was deleted
+             */
+            userId: {
+                type: DataTypes.INTEGER,
+                allowNull: true
+            },
```

Comment:
My plan is to just manually add this column to the test/live tables – given it allows NULL it's a safe change (just done it locally) even before we merge this.